specs/TOOL-035-gitea-repository-reconciliation/verification.md

```
@@ -209,10 +209,33 @@ make provision NODE=bee ENV=prod TAGS=gitea
 Idempotent, and the re-mint is gated on the SOPS key being absent — so the rotation is what opens the
 gate, and a provision with the key present does nothing.
 
-**AC2 is structural rather than observed, deliberately.** The forge currently holds no undeclared
-repository, so there is no stray to report. `test_the_plan_has_no_deletion_field` asserts over
-`dataclasses.fields(ReconcilePlan)` that no field a deletion could travel in exists at all, which
-holds for every future caller — a stronger claim than a fixture that happened not to delete anything.
+**AC2 — observed as well as structural (2026-09-02).** The forge held no stray, so one was made:
+`personal/zz-stray`, created by the bot, empty, undeclared. Both paths that could have removed it were
+then run against it:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)